dcspy/logitech.py

```diff
@@ -164,11 +163,7 @@ def check_buttons(self) -> LcdButton:
         """
         for btn in self.buttons:
             if lcd_sdk.logi_lcd_is_button_pressed(btn.value):
-                if not self.lcdbutton_pressed:
```

Comment:
This looks to have been introduced a long time ago to guard around double G-key or LCD button presses. With the separation of the G-key SDK handler I don't know if this is still relevant or not. With the change here now if the LCD button is pressed then we send the request for this button.